legipy/parsers/law_parser.py

```diff
@@ -37,7 +37,7 @@ def parse_law(url, html, id_legi):
         try:
             LAW_KINDS.index(prop.group(2))
             law.kind = prop.group(2)
-        except:
+        except (IndexError, ValueError, TypeError):
```

Comment:
Thanks for the update, here also it would need a change before I can merge the PR

Reply:
Thanks, done!